test/functional/apps/getting_started/_shakespeare.js

```diff
@@ -47,7 +47,6 @@ export default function ({ getService, getPageObjects }) {
   ]);
 
   // https://www.opensearch.org/guide/en/kibana/current/tutorial-load-dataset.html
```

Comment:
This link doesn't exist either. In this case we can probably remove it entirely?

Reply:
it looks like I add it back accidentally in this new revision. will remove thanks